.aegir.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports = {
+  webpack: {
+    node: {
+      path: true,
+
+      Buffer: true
```

Comment:
do we need this?

Reply:
Yes, I've updated the file with comments about which modules require the node libs